src/imdb_utils.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+import kagglehub
+import os
+import shutil
+
+resources_path = "../resources"
+file_name = "movie_metadata.csv"
+kaggle_data = "carolzhangdc/imdb-5000-movie-dataset"
+
+def aquireIMDbDataFrame():
```

Comment:
can we use more conventional names like `getIMDbDataFrame()`

Reply:
"get" would be used if the item is readily available without work, such as in memory store.
"acquire" would be used if the process of obtaining the item is involved, like a network request.
but if you want it to be "get" fine with me.   We can talk tonight

Reply:
I am ok with this rationale